src/components/Pages/addOrder.jsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { Button } from "../Button/Button";
+import { InputField } from "../InputField/InputField";
+//import "./AddOrder.css";
+
+export const AddOrder = () => {
+  return (
+    <div className="AddOrder">
+      <h1 className="App-header">Add New Order</h1>
+      <form>
+        <InputField
+          label={"Order ID"}
+          placeholder={"A78D4S5S5"}
+          id={"transaction_id"}
+        />
+        <InputField
+          label={"Member ID"}
+          id={"member_id"}
+          type={"email"}
+          placeholder={"alfie.santos@gmail.com"}
+        />
+        <Button buttonType={"primary"} isDisabled={true}>
+          Add Order
+        </Button>
+      </form>
+    </div>
+  );
+};
```

Comment:
* import CSS is commented out
* The placeholders don't match the ones set in Figma
* You should swap ID to Name. InputField doesn't take an ID value and for the req.body.(thingy) I believe this uses the Name attribute to refer to the values submitted in the form.

Reply:
Also for the InputFields we are only using Date and Text types, so the Member ID field should just be Text. In Figma is does say to enter an email or mobile number, I feel like we've discussed about only using one or the other, could be discussing that with everyone again...

Reply:
Ah I see where you got the placeholders now, those are example user inputs, not the placeholders for the fields